.github/workflows/ci.yaml

```diff
@@ -166,6 +166,14 @@ jobs:
         #  testlib modules of gax
         run: mvn package clirr:check -DskipTests
 
+  requirements-kokoro:
+    runs-on: ubuntu-latest
+    container: gcr.io/cloud-devrel-public-resources/java11
+    steps:
+      - uses: actions/checkout@v3
+      - run: python3 --version
+      - run: python3 -m pip install --require-hashes -r .kokoro/requirements.txt
```

Comment:
Great. Can you also add check to confirm `releasetool` module import? 

- https://github.com/googleapis/gapic-generator-java/blob/main/.kokoro/release/stage.sh#LL23C3-L23C25
- https://github.com/googleapis/synthtool/blob/master/docker/owlbot/java/container_test.yaml#L36